wp-content/themes/core/theme.json

```diff
@@ -436,7 +431,8 @@
 							"fontStyle": "normal"
 						}
 					}
-				}
+				},
+				"css": "> blockquote p {font-size: var(--wp--preset--font-size--50); font-weight: var(--wp--custom--font-weight--bold); line-height: 1.2;}"
```

Comment:
🔴 yea, let's not do this. I know @dpellenwood wasn't a fan and has commented on it before. Let's move the styles to our block's `.pcss` partial